packages/otel/README.md

```diff
@@ -9,7 +9,7 @@
 ## 📦 Installation
 
 ```sh
-npm install @vercel/otel
+npm install @vercel/otel @opentelemetry/api @opentelemetry/api-logs @opentelemetry/instrumentation @opentelemetry/resources @opentelemetry/sdk-logs @opentelemetry/sdk
```

Comment:
Why are these needed? IIUC:

* `@opentelemetry/api` and `@opentelemetry/api-logs` get in as peer deps
* `@opentelemetry/instrumentation` and `@opentelemetry/sdk/sdk-logs` shouldn't be needed at all.
* `@opentelemetry/resources` - I don't think this one is needed either

Reply:
@dvoytenko The dependencies I added here are listed as [peerDependencies](https://github.com/vercel/otel/blob/main/packages/otel/package.json#L92-L100) listed in the `package.json`.

When using pnpm to install `@vercel/otel`, they are missing from `node_modules` (check the screenshot below)
![CleanShot 2024-11-14 at 16 33 33@2x](https://github.com/user-attachments/assets/ac1bee29-1471-49a1-8f00-b2e46ae1ac67)